addons/web/models/ir_http.py

```diff
@@ -63,7 +65,7 @@ def session_info(self):
             qweb_checksum = HomeStaticTemplateHelpers.get_qweb_templates_checksum(debug=request.session.debug, bundle="web.assets_qweb")
             lang = user_context.get("lang")
             translation_hash = request.env['ir.translation'].get_web_translations_hash(mods, lang)
-            menu_json_utf8 = json.dumps(request.env['ir.ui.menu'].load_menus(request.session.debug), default=ustr, sort_keys=True).encode()
+            menu_json_utf8 = json.dumps(request.env['ir.ui.menu'].load_menus(request.session.debug), default=ustr).encode()
```

Comment:
That seems like it's likely to cause caching issues (cache misses): `sort_keys` ensures the JSON serialization is completely deterministic rather than in whatever order the dicts are iterated, was there a specific reason to remove it?